.github/workflows/slack_notify.yml

```diff
@@ -0,0 +1,27 @@
+name: Slack notification for Github Actions
+on:
+  workflow_run:
+    workflows:
+      - Push to main
+      - release
+    types:
+      - completed
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Announce result
```

Comment:
Can you add another job if the status of the job is a failure? Something like: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow

Reply:
Discussed in slack :)
- This is going to be done in the second phase of slack notifications :) as well as other details